cmd/bundle/main.go

```diff
@@ -16,19 +17,22 @@ import (
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/name"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
-	"github.com/shipwright-io/build/pkg/bundle"
 	"github.com/spf13/pflag"
+
+	"github.com/shipwright-io/build/pkg/bundle"
 )
 
 var flagValues struct {
-	image      string
-	target     string
-	secretPath string
+	image       string
+	target      string
+	secretPath  string
+	imageDigest string
 }
 
 func init() {
 	pflag.StringVar(&flagValues.image, "image", "", "Location of the bundle image (mandatory)")
 	pflag.StringVar(&flagValues.target, "target", "/workspace/source", "The target directory to place the code")
+	pflag.StringVar(&flagValues.imageDigest, "result-file-image-digest", "", "A file to write the image digest")
```

Comment:
```suggestion
	pflag.StringVar(&flagValues.resultFileImageDigest, "result-file-image-digest", "", "A file to write the image digest")
```

Needs updates at other places.